### PR TITLE
[Reference cleanup]LPS-162280

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to convert method references to field references in export-import-service| Part 1.
Thanks!
https://issues.liferay.com/browse/LPS-162280

### CODE REVIEW

**Comment by JanisZhang** [comment]
Hi @catarineaguiar , Could you please help me scan the test result? thanks.

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutImportController.java [hunk 1]
```diff
@@ -468,7 +468,7 @@ protected void validateFile(
 			throw new LARFileException(LARFileException.TYPE_MISSING_MANIFEST);
 		}
 
-		Element rootElement = null;
+		Element rootElement;
```
Comment (dantewang) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutImportController.java:471]: @JanisZhang Same question as the other PR..

Comment (JanisZhang) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutImportController.java]: @dantewang Noted, thanks.

**Comment by JanisZhang** [comment]
Hi @dantewang, force-pushed.

Hi @catarineaguiar,
I only drop one SF commit. It shouldn't affect the ci results. Thanks

**Review by dantewang — APPROVED** [review]

**Comment by catarineaguiar** [comment]
@JanisZhang no regressions found.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Pull request closed without merging**